src/main/java/jackyy/dimensionaledibles/registry/ModConfig.java

```diff
@@ -30,6 +30,10 @@
         public boolean overworldApple = true;
         @Config.Comment("Set to true to enable Custom Cake.")
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
I think you misunderstand what these are: nested Annotation classes defined by the Forge API. We have no control over that code and merely call them as required for correct integration with Forge config file persistence.

Reply:
I’m fairly certain this is a bot, based on their Github history. Gregicality has had similar reviews like this from the same account, and they were blocked.

Reply:
~400 reviews in a day, must be a bot